Aspire Budgeting/Facilities/Images.swift

```diff
@@ -0,0 +1,12 @@
+//
+// Images.swift
+// Aspire Budgeting
+//
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/080f65df2311b45c8d4b592dff399d3b1bb01b5f/Aspire Budgeting/Facilities/Images.swift#L6"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>Images.swift:6</code></td>
    </tr>
  </tbody>
</table>